packages/pymedphys_pinnacle/src/pymedphys_pinnacle/export/image.py

```diff
@@ -0,0 +1,268 @@
+                i + 1) * int(image_header["x_dim"]) * int(image_header["y_dim"])]
+            allframeslist.append(frame_array)
+    image.logger.debug("Length of frames list: " + str(len(allframeslist)))
+    image.logger.debug(image_info[0])
+
+    curframe = 0
+    for info in image_info:
+        sliceloc = -info['TablePosition'] * 10
+        instuid = info['InstanceUID']
+        seriesuid = info['SeriesUID']
+        classuid = info['ClassUID']
+        frameuid = info['FrameUID']
+        studyinstuid = info['StudyInstanceUID']
+        slicenum = info['SliceNumber']
+
+        dateofscan = image_set['scan_date']
+        timeofscan = image_set['scan_time']
+
+        file_meta = Dataset()
+        file_meta.MediaStorageSOPClassUID = classuid
+        file_meta.MediaStorageSOPInstanceUID = instuid
+        file_meta.TransferSyntaxUID = GTransferSyntaxUID
+        # this value remains static since implementation for creating
+        # file is the same
+        file_meta.ImplementationClassUID = GImplementationClassUID
+
+        image_file_name = modality + '.' + instuid + '.dcm'
+        ds = FileDataset(image_file_name, {},
+                         file_meta=file_meta, preamble=b'\x00' * 128)
+
+        ds.SpecificCharacterSet = "ISO_IR 100"
+        ds.ImageType = ['ORIGINAL', 'PRIMARY', 'AXIAL']
+        ds.AccessionNumber = ''
+        ds.SOPClassUID = classuid
+        ds.SOPInstanceUID = instuid
+        ds.StudyDate = dateofscan
+        ds.SeriesDate = dateofscan
+        ds.AcquisitionDate = dateofscan
+        ds.ContentDate = dateofscan
+        ds.AcquisitionTime = timeofscan
+        ds.Modality = modality
+        # This should come from Manufacturer in header, but for some
+        # patients it isn't set??
+        ds.Manufacturer = "GE MEDICAL SYSTEMS"
+        ds.StationName = modality
+        ds.PatientsName = patient_info['FullName']
+        ds.PatientID = patient_info['MedicalRecordNumber']
+        ds.PatientsBirthDate = patient_info['DOB']
+        ds.BitsAllocated = 16
+        ds.BitsStored = 16
+        ds.HighBit = 15
+        ds.PixelRepresentation = 1
+        ds.RescaleIntercept = -1024
+        ds.RescaleSlope = 1.0
+        # ds.kvp = ?? This should be peak kilovoltage output of x ray
+        # generator used
+        ds.PatientPosition = currentpatientposition
+        # this is probably x_pixdim * xdim = y_pixdim * ydim
+        ds.DataCollectionDiameter = float(
+            image_header["x_pixdim"]) * 10 * float(image_header["x_dim"])
+        ds.SpatialResolution = 0.35  # ???????
+        # ds.DistanceSourceToDetector = #???
+        # ds.DistanceSourceToPatient = #????
+        ds.GantryDetectorTilt = 0.0  # ??
+        ds.TableHeight = -158.0  # ??
+        ds.RotationDirection = "CW"  # ???
+        ds.ExposureTime = 1000  # ??
+        ds.XRayTubeCurrent = 398  # ??
+        ds.GeneratorPower = 48  # ??
+        ds.FocalSpots = 1.2  # ??
```

Comment:
I do think these hard coded results need to be removed as @Matthew-Jennings mentioned in a previous comment.